data-raw/watershed/regional_approximation/Regional_Approximation.Rmd

```diff
@@ -0,0 +1,168 @@
+model_flow_summary(cvpiaHabitat::cow_creek_instream)
+
+
+# create list of wua (sq ft/1000 ft) approximators for each watershed in region
+get_approx_spwn <- function(df) {approxfun(df$flow_cfs, df$FR_spawn_wua, rule = 2)}
+get_approx_fry <- function(df) {approxfun(df$flow_cfs, df$FR_fry_wua, rule = 2)}
+get_approx_juv <- function(df) {approxfun(df$flow_cfs, df$FR_juv_wua, rule = 2)}
+
+upmidsac_spwn <- list(get_approx_spwn(cvpiaHabitat::battle_creek_instream),
+                     get_approx_spwn(cvpiaHabitat::butte_creek_instream),
+                     get_approx_spwn(cvpiaHabitat::clear_creek_instream))
+
+upmidsac_juv <- list(get_approx_juv(cvpiaHabitat::battle_creek_instream),
+                     get_approx_juv(cvpiaHabitat::butte_creek_instream),
+                     get_approx_juv(cvpiaHabitat::clear_creek_instream),
+                     get_approx_juv(cvpiaHabitat::cow_creek_instream))
+
+upmidsac_fry <- list(get_approx_fry(cvpiaHabitat::battle_creek_instream),
+                     get_approx_fry(cvpiaHabitat::butte_creek_instream),
+                     get_approx_fry(cvpiaHabitat::clear_creek_instream),
+                     get_approx_fry(cvpiaHabitat::cow_creek_instream))
+
+cvpiaFlow::flows_cfs %>%
+  gather(watershed, flow, -date) %>%
+  filter(watershed %in% watersheds_without_modeling, month(date) %in% 1:8) %>%
+  group_by(watershed) %>%
+  summarise(min = min(flow), median = median(flow), mean = mean(flow),
+            q90 = quantile(flow, .9), max = max(flow))
+
+cvpiaFlow::flows_cfs %>%
+  gather(watershed, flow, -date) %>%
+  filter(watershed %in% watersheds_with_modeling, month(date) %in% 1:8) %>%
+  group_by(watershed) %>%
+  summarise(min = min(flow), median = median(flow), mean = mean(flow),
+            q90 = quantile(flow, .9), max = max(flow))
+
+flows <- cvpiaHabitat::clear_creek_instream$flow_cfs
+
+upper_mid_sac_region_instream <- purrr::map_df(flows, function(flow) {
+  wua_spn <- mean(purrr::map_dbl(1:length(upmidsac_spwn), function(i){upmidsac_spwn[[i]](flow)}))
+  wua_fry <- mean(purrr::map_dbl(1:length(upmidsac_fry), function(i){upmidsac_fry[[i]](flow)}))
+  wua_juv <- mean(purrr::map_dbl(1:length(upmidsac_juv), function(i){upmidsac_juv[[i]](flow)}))
+  tibble(flow_cfs = flow, FR_spawn_wua = wua_spn, FR_fry_wua = wua_fry, FR_juv_wua = wua_juv,
+         watershed = 'Upper-mid Sacramento River Region')
+})
+#usethis::use_data(upper_mid_sac_region_instream, overwrite = TRUE)
+```
+
+```{r, echo = FALSE}
+knitr::kable(align = 'c', head(upper_mid_sac_region_instream, 5))
+```
+
+*...with 18 more rows.*
+
+### Spawning WUA 
+The instream spawning habitat weighted usable areas for un-modeled watersheds in the Upper-mid Sacramento River Region.
+
+#### Spawning WUA Plot
+The following plot shows the weighted usable spawning area in square feet per thousand feet of river for Fall Run Chinook Salmon. These area per length rates are multiplied by the total spawning reach length mapped by the SIT. 
```

Comment:
"...for Fall Run Chinook Salmon" plz change to "for un-modeled Upper Sacramento River tributaries Chinook Salmon"